From edcfea31a3381decc13f4550e412e5d84b8e968b Mon Sep 17 00:00:00 2001
From: Biswapriyo Nath <nathbappai@gmail.com>
Date: Thu, 1 Feb 2024 19:39:48 +0530
Subject: [PATCH] adb include missing headers

---
 adb_mdns.cpp  | 1 +
 sysdeps/uio.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/adb_mdns.cpp b/adb_mdns.cpp
index dde08d7..312ea3c 100644
--- a/adb_mdns.cpp
+++ b/adb_mdns.cpp
@@ -19,6 +19,7 @@
 #include "adb_mdns.h"
 
 #include <algorithm>
+#include <atomic>
 #include <set>
 
 #include <android-base/stringprintf.h>
diff --git a/sysdeps/uio.h b/sysdeps/uio.h
index f3ace64..266a20b 100644
--- a/sysdeps/uio.h
+++ b/sysdeps/uio.h
@@ -34,6 +34,7 @@ ssize_t adb_writev(borrowed_fd fd, const adb_iovec* iov, int iovcnt);
 
 #else
 
+#include <limits.h>
 #include <sys/uio.h>
 using adb_iovec = struct iovec;
 inline ssize_t adb_writev(borrowed_fd fd, const adb_iovec* iov, int iovcnt) {
-- 
2.55.0

