From fc345bb6828cf37d5faa1fb2ae9c74d530d251ca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 1 Sep 2019 15:05:17 +0000
Subject: [PATCH] fdevent: missing atomic include

---
 fdevent/fdevent.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fdevent/fdevent.h b/fdevent/fdevent.h
index bb3af746..b4dfc8a2 100644
--- a/fdevent/fdevent.h
+++ b/fdevent/fdevent.h
@@ -28,6 +28,7 @@
 #include <optional>
 #include <unordered_map>
 #include <variant>
+#include <atomic>
 
 #include <android-base/thread_annotations.h>
 
