From 0eddc30a5913401262df2830b28e8dadaf1d0c26 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 27 Feb 2018 17:48:49 +0100
Subject: [PATCH] Define clang-only nullability specifiers when __clang__ is
 not defined

---
 adb_mdns.h | 6 ++++++
 sysdeps.h  | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/adb_mdns.h b/adb_mdns.h
index a015f53..0e8fc4e 100644
--- a/adb_mdns.h
+++ b/adb_mdns.h
@@ -21,6 +21,12 @@
 #include <string>
 #include <string_view>
 
+// Clang-only nullability specifiers
+#ifndef __clang__
+#define _Nonnull
+#define _Nullable
+#endif
+
 // The rules for Service Names [RFC6335] state that they may be no more
 // than fifteen characters long (not counting the mandatory underscore),
 // consisting of only letters, digits, and hyphens, must begin and end
diff --git a/sysdeps.h b/sysdeps.h
index 5ba85b4d..2115a32b 100644
--- a/sysdeps.h
+++ b/sysdeps.h
@@ -55,6 +55,12 @@ static inline void* mempcpy(void* dst, const void* src, size_t n) {
 
 std::optional<ssize_t> network_peek(borrowed_fd fd);
 
+// Clang-only nullability specifiers
+#ifndef __clang__
+#define _Nonnull
+#define _Nullable
+#endif
+
 #ifdef _WIN32
 
 #include <ctype.h>
