From f57234b135e29fc69ec44823e17e19941e3be0cd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 1 Sep 2019 14:22:06 +0000
Subject: [PATCH] Include cstring in various file missing it

---
 crypto/x509_generator.cpp | 1 +
 types.h                   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/crypto/x509_generator.cpp b/crypto/x509_generator.cpp
index 43b81530..8a0cf607 100644
--- a/crypto/x509_generator.cpp
+++ b/crypto/x509_generator.cpp
@@ -16,6 +16,7 @@
 
 #include "adb/crypto/x509_generator.h"
 
+#include <cstring>
 #include <vector>
 
 #include <android-base/logging.h>
diff --git a/types.h b/types.h
index 620aa8eb..e263651e 100644
--- a/types.h
+++ b/types.h
@@ -22,6 +22,7 @@
 #include <memory>
 #include <utility>
 #include <vector>
+#include <cstring>
 
 #include <android-base/logging.h>
 
