From dc91693fd1c4190fcae4c681a401118e33028480 Mon Sep 17 00:00:00 2001
From: jershell <jershell@mail.ru>
Date: Mon, 6 Jun 2022 16:29:40 +0300
Subject: [PATCH] fix for Fix-compilation-with-libusb

---
 client/usb_libusb_device.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/usb_libusb_device.cpp b/client/usb_libusb_device.cpp
index dd51963..d5d162d 100644
--- a/client/usb_libusb_device.cpp
+++ b/client/usb_libusb_device.cpp
@@ -459,7 +459,7 @@ bool LibUsbDevice::OpenDeviceHandle() {
 
     int rc = libusb_open(device_, &device_handle_);
     if (rc != 0) {
-        VLOG(USB) << "Unable to open device: " << GetSerial() << " :" << libusb_strerror(rc);
+        VLOG(USB) << "Unable to open device: " << GetSerial() << " :" << libusb_strerror(static_cast<libusb_error>(rc));
         return false;
     }
 
-- 
2.53.0

