From d134b3e89ff11b3a0bc1efa906dd4e0c6a3171a4 Mon Sep 17 00:00:00 2001
From: Anatol Pomozov <anatol.pomozov@gmail.com>
Date: Fri, 25 Aug 2017 23:30:34 +0000
Subject: [PATCH 07/12] Include libusb-1.0/libusb.h instead of libusb/libusb.h

---
 adb/client/usb_libusb.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adb/client/usb_libusb.cpp b/adb/client/usb_libusb.cpp
index 81201995a..c3b9c543e 100644
--- a/adb/client/usb_libusb.cpp
+++ b/adb/client/usb_libusb.cpp
@@ -28,7 +28,7 @@
 #include <thread>
 #include <unordered_map>
 
-#include <libusb/libusb.h>
+#include <libusb-1.0/libusb.h>
 
 #include <android-base/file.h>
 #include <android-base/logging.h>
-- 
2.17.1

