From e83f1e48b02652089d75a00e0ee9ea49cbf940cc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 27 Feb 2018 17:49:52 +0100
Subject: [PATCH 07/11] Include libusb-1.0/libusb.h instead of libusb/libusb.h

---
 adb/client/usb_libusb.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adb/client/usb_libusb.cpp b/adb/client/usb_libusb.cpp
index 81201995a..c3b9c543e 100644
--- a/adb/client/usb_libusb.cpp
+++ b/adb/client/usb_libusb.cpp
@@ -28,7 +28,7 @@
 #include <thread>
 #include <unordered_map>
 
-#include <libusb/libusb.h>
+#include <libusb-1.0/libusb.h>
 
 #include <android-base/file.h>
 #include <android-base/logging.h>
-- 
2.17.1

