From 7f460be2f974987faee0a8776ce3725b84ff28d3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 27 Feb 2018 17:51:52 +0100
Subject: [PATCH 08/11] Quote ADB_VERSION in adb.cpp

---
 adb/adb.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adb/adb.cpp b/adb/adb.cpp
index 6b30be884..04ba85c2d 100644
--- a/adb/adb.cpp
+++ b/adb/adb.cpp
@@ -67,7 +67,7 @@ std::string adb_version() {
         "Android Debug Bridge version %d.%d.%d\n"
         "Version %s\n"
         "Installed as %s\n",
-        ADB_VERSION_MAJOR, ADB_VERSION_MINOR, ADB_SERVER_VERSION, ADB_VERSION,
+        ADB_VERSION_MAJOR, ADB_VERSION_MINOR, ADB_SERVER_VERSION, "ADB_VERSION",
         android::base::GetExecutablePath().c_str());
 }
 
-- 
2.17.1

