From f27701927b9ee580848127b6d5fb69cfc9601c84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 28 Feb 2018 20:22:09 +0100
Subject: [PATCH 09/13] socket.h: include sys/select.h for fd_set used in
 socket.cpp

---
 fastboot/socket.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fastboot/socket.h b/fastboot/socket.h
index 7eaa0ab2c..a772b1227 100644
--- a/fastboot/socket.h
+++ b/fastboot/socket.h
@@ -39,6 +39,7 @@
 #include <utility>
 #include <vector>
 
+#include <sys/select.h>
 #include <android-base/macros.h>
 #include <cutils/sockets.h>
 #include <gtest/gtest_prod.h>
-- 
2.17.1

