From 697ea0147bd28082cc834b37efb15b4da4540447 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 21 Aug 2019 19:59:43 +0200
Subject: [PATCH] libcutils: include sys/select.h for timeval

---
 libcutils/sockets_unix.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libcutils/sockets_unix.cpp b/libcutils/sockets_unix.cpp
index 6acdcd89b..32ab56080 100644
--- a/libcutils/sockets_unix.cpp
+++ b/libcutils/sockets_unix.cpp
@@ -20,6 +20,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/socket.h>
+#include <sys/select.h>
 #include <sys/uio.h>
 #include <sys/un.h>
 #include <time.h>
