From b703886ebd0f6e17e642081be956f2d834c9478b Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Fri, 16 Jan 2026 12:56:20 +0100
Subject: [PATCH] zip_writer: Add <cstdint> include

---
 include/ziparchive/zip_writer.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/ziparchive/zip_writer.h b/include/ziparchive/zip_writer.h
index 268e8b6..aca7f4e 100644
--- a/include/ziparchive/zip_writer.h
+++ b/include/ziparchive/zip_writer.h
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <cstdio>
 #include <ctime>
 
-- 
2.52.0

