From ca96ebc1cc16ae246e28c98f71e39d0d8b23243a Mon Sep 17 00:00:00 2001
From: Biswapriyo Nath <nathbappai@gmail.com>
Date: Wed, 15 Feb 2023 23:46:43 +0530
Subject: [PATCH] fix cxx narrowing error

---
 zip_archive.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zip_archive.cc b/zip_archive.cc
index e392a20..c5b5ae8 100644
--- a/zip_archive.cc
+++ b/zip_archive.cc
@@ -1370,7 +1370,7 @@ bool Reader::IsZeroCopy() const {
 }  // namespace zip_archive
 
 static std::span<uint8_t> bufferToSpan(zip_archive::Writer::Buffer buf) {
-  return {buf.first, ssize_t(buf.second)};
+  return {buf.first, size_t(buf.second)};
 }
 
 template <bool OnIncfs>
-- 
2.39.2

