From 0287fb7244568a5f08c0ac153dcb464df67ff140 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 27 Feb 2018 17:37:12 +0100
Subject: [PATCH] fix "initializer element is not constant"-error when building
 with g++

---
 liblog/logprint.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/liblog/logprint.cpp b/liblog/logprint.cpp
index a5c5edd0..71868a35 100644
--- a/liblog/logprint.cpp
+++ b/liblog/logprint.cpp
@@ -884,9 +884,9 @@ static int android_log_printBinaryEvent(const unsigned char** pEventData, size_t
           }
           break;
         case TYPE_MONOTONIC: {
-          static const uint64_t minute = 60;
-          static const uint64_t hour = 60 * minute;
-          static const uint64_t day = 24 * hour;
+          #define minute (60)
+          #define hour (60 * minute)
+          #define day (24 * hour)
 
           /* Repaint as unsigned seconds, minutes, hours ... */
           outBuf -= outCount;
