From aba6bdbbe26285c5bee14c88a514995e4c0e99d1 Mon Sep 17 00:00:00 2001
From: MoetaYuko <loli@yuko.moe>
Date: Fri, 10 Dec 2021 17:36:31 +0800
Subject: [PATCH] libjsonpbparse: Fix build after protobuf util::Status
 refactorization

---
 libjsonpb/parse/jsonpb.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/libjsonpb/parse/jsonpb.cpp b/libjsonpb/parse/jsonpb.cpp
index 3a042e7..62d651d 100644
--- a/libjsonpb/parse/jsonpb.cpp
+++ b/libjsonpb/parse/jsonpb.cpp
@@ -48,7 +48,13 @@ ErrorOr<std::string> MessageToJsonString(const Message& message) {
                                    &json, options);
 
   if (!status.ok()) {
+#if GOOGLE_PROTOBUF_VERSION < 3016000
+    return MakeError<std::string>(status.error_message().as_string());
+#elif GOOGLE_PROTOBUF_VERSION < 4022000
+    return MakeError<std::string>(status.message().as_string());
+#else
     return MakeError<std::string>(std::string(status.message()));
+#endif
   }
   return ErrorOr<std::string>(std::move(json));
 }
@@ -61,7 +67,13 @@ ErrorOr<std::monostate> JsonStringToMessage(const std::string& content, Message*
   std::string binary;
   auto status = JsonToBinaryString(resolver.get(), GetTypeUrl(*message), content, &binary);
   if (!status.ok()) {
+#if GOOGLE_PROTOBUF_VERSION < 3016000
+    return MakeError<std::monostate>(status.error_message().as_string());
+#elif GOOGLE_PROTOBUF_VERSION < 4022000
+    return MakeError<std::monostate>(status.message().as_string());
+#else
     return MakeError<std::monostate>(std::string(status.message()));
+#endif
   }
   if (!message->ParseFromString(binary)) {
     return MakeError<std::monostate>("Fail to parse.");
