From 70d3294a12f05aea953cae415810345f516d778d Mon Sep 17 00:00:00 2001
From: MoetaYuko <loli@yuko.moe>
Date: Fri, 10 Dec 2021 17:43:17 +0800
Subject: [PATCH] lpadd: Add missing header for strerror

---
 partition_tools/lpadd.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/partition_tools/lpadd.cc b/partition_tools/lpadd.cc
index 48cad72..2d19059 100644
--- a/partition_tools/lpadd.cc
+++ b/partition_tools/lpadd.cc
@@ -16,6 +16,7 @@
 #include <sysexits.h>
 #include <unistd.h>
 
+#include <cstring>
 #include <iostream>
 #include <optional>
 
